zephyr/ra/ra_cfg/fsp_cfg/r_glcdc_cfg.h

```diff
@@ -0,0 +1,24 @@
+/*
+* Copyright (c) 2020 - 2024 Renesas Electronics Corporation and/or its affiliates
+*
+* SPDX-License-Identifier: BSD-3-Clause
+*/
+
+#ifndef R_GLCDC_CFG_H_
+#define R_GLCDC_CFG_H_
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define GLCDC_CFG_PARAM_CHECKING_ENABLE      (BSP_CFG_PARAM_CHECKING_ENABLE)
+#define GLCDC_CFG_COLOR_CORRECTION_ENABLE    (false)
+
+/* Enable DSI function handling */
+#if 0
+ #define GLCDC_CFG_USING_DSI
+#endif
```

Comment:
@thenguyenyf : Please recheck this config whether it is necessary or not.

Reply:
I fixed it. `GLCDC_CFG_USING_DSI` will be undef because Zephyr MIPI driver is not internal controlled by r_glcdc